packages/slap/slap.3.0.1/opam

```diff
@@ -13,17 +13,20 @@ build: [
 ]
 install: ["ocaml" "setup.ml" "-install"]
 remove: [
-  ["ocamlfind" "remove" "slap"]
+  ["ocaml" "setup.ml" "-configure"
```

Comment:
`setup.ml` is no longer present (the tarball extracted files having been removed), so that will not work.
See https://travis-ci.org/ocaml/opam-repository/jobs/133722294#L654